docs/pages/application-access/guides/aws-console.mdx

```diff
@@ -20,8 +21,49 @@ This guide will explain how to:
 - Teleport with Application Access. Follow [Getting Started](../getting-started.mdx)
   or [Connecting Apps](./connecting-apps.mdx) guides to get it running.
 - IAM permissions in the AWS account you want to connect.
+- AWS EC2 or other instance where you can assign a IAM Security Role for the Teleport Agent
 
-## Step 1. Update IAM roles trust relationships
+
+## Step 1. Configure Read Only and Power User Roles
+
```

Comment:
Do you think a note would be useful that would say something like "if you already have IAM role you want your users to impersonate, you can skip this step"?

Reply:
added comment